### DISCUSSION TITLE
Is it possible to use LoRa tu customize model

### DISCUSSION THREAD

**Discussion by landsolar** [upscayl/upscayl#614] [Q&A]
Hi, 

This Lora is for eye iris texture : https://civitai.com/models/87994/airis-eye-texture
Is it possible to use it to customize Upscayl ?

Or may be I need a custom model ?

**Comment by NayamAmarshe** [comment]
No, not possible. That is a stable diffusion model. Upscayl uses REAL-ESRGAN.
